runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TimeTrackingConfig.java

```diff
@@ -83,6 +83,29 @@ default int defaultTimerMinutes()
 		return 5;
 	}
 
+	@ConfigItem(
+		keyName = "sortOrder",
+		name = "Sort Order",
+		description = "The order in which to sort the timers",
+		position = 5
+	)
+	default SortOrder sortOrder()
+	{
+		return SortOrder.NONE;
+	}
+
+	@ConfigItem(
+		keyName = "timerWarningSeconds",
+		name = "Timer Warning (Seconds)",
+		description = "The time at which to change the timer color to the warning color",
+		position = 6
+	)
+	@Units(Units.SECONDS)
+	default int timerWarningSeconds()
```

Comment:
Now that it's using `Units` the name `Timer Warning (Seconds)` feels redundant, consider renaming the option and corresponding fields to something like `Timer Warning Threshold` or similar